### DISCUSSION TITLE
Dialysis

### DISCUSSION THREAD

**Discussion by limyzd** [MIT-LCP/mimic-code#1197] [MIMIC-IV]
Is there any means to determine patients who are receiving dialysis in mimic-iv?
Can we distinguish the various modes of dialysis - conventional haemodialysis, SLED, CRRT?

**Comment by alistairewj** [comment]
Yep - they document it in the ICU chart

- they explicitly document continuous forms of dialysis in *procedureevents*
- various settings of continuous dialysis are documented in *chartevents*
- the output from intermittent hemodialysis is documented in *outputevents*

We have a derived view here: https://github.com/MIT-LCP/mimic-code/blob/main/mimic-iv/concepts/treatment/rrt.sql

**Comment by fysqy** [comment]
Is there any way to know whether a patient received dialysis in a ward before admission or after discharge from the ICU?

**Reply by alistairewj** [reply]
You could try the *poe* table - it might have orders for dialysis in it.
